assets/stylesheets/base/_utilities.scss

```diff
@@ -402,11 +402,16 @@ a.underline,
   margin-left: 5px;
 }
 
-// Square corners
+// Border Radius
+
 .square-corners {
   border-radius: 0;
 }
 
+.circular {
+  border-radius: 100%;
+}
```

Comment:
Here's an instance where I've added a utility so it could be used elsewhere. In cases like this, should I also be documenting this in the DDK?